lib/vscode_helper.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'dart:convert';
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/convert.dart';
+import 'package:flutter_tools/src/globals.dart' as globals;
+import 'package:flutter_tools/src/project.dart';
+
+const String kConfigNameAttach = 'Attach (flutter-tizen)';
+
+final RegExp _commentFormat =
+    RegExp(r'(:?)(/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+/|//.*\n?)');
+
+String _removeComments(String jsonString) {
+  return jsonString.replaceAllMapped(
+    _commentFormat,
+    (Match match) {
+      // "aaa://bbb" contains "//" but is not a comment.
+      return match.group(1) == ':' ? match.group(0) : '';
+    },
+  ).trim();
+}
+
+void updateLaunchJsonFile(FlutterProject project, Uri observatoryUri) {
+  Directory vscodeDir;
+  String cwd = r'${workspaceFolder}';
+  if (project.directory.basename == 'example') {
+    final FlutterProject parentProject =
+        FlutterProject.fromDirectory(project.directory.parent);
+    if (parentProject.isPlugin || parentProject.isModule) {
+      vscodeDir = parentProject.directory.childDirectory('.vscode')
+        ..createSync(recursive: true);
+      cwd += '/example';
+    }
+  }
```

Comment:
I think the `launch.json` file should be just located in  current project. Because users run and debug the example project itself, not a parent module or plugin. In general, users open the example project to debug it.